source/ceylon/file/internal/createTemporaryDirectory.ceylon

```diff
@@ -0,0 +1,13 @@
+import ceylon.file {
+    Directory
+}
+
+import java.nio.file {
+    JFiles=Files
+}
+
+shared Directory createTemporaryDirectory(
```

Comment:
ok, I'll merge it into the `TemporaryDirectory` initializer, and the same for `createTemporaryFile`.